### PR TITLE
Added Triangle shape

### PR DESCRIPTION
**Pull request by judy-n**

### CODE REVIEW

**Comment by ashutosh1919** [comment]
@judy-n Thank you so much for raising PR. Let this PR stay open for 3-4 days. Since we have released the project yesterday, we want to stabilize the traffic as of now. I will start reviewing PRs starting from Monday (1st March) 💯

File: src/components/shapesdemos.tsx [hunk 1]
```diff
@@ -55,6 +55,14 @@ const polygoncardExample = `
 />
 `;
 
+const triangleExample = `
+<Triangle
+    size="150px"
+    zIndex={2}
+    color="linear-gradient(135deg, #f9a8d4, #eab309)"
```
Comment (ashutosh1919) [inline src/components/shapesdemos.tsx:62]:
```suggestion
    color="linear-gradient(135deg, #fde047, #eab308)"
```

We are following [tailwind colors](https://tailwindcss.com/docs/customizing-colors) so it is good to stay consistent with the gradient theme which we have used in other components.

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
@judy-n , Thanks for contribution.

Below are some of the issues that you can fix before we can merge. Other than below issues, there is one more thing that you need to add.

In [Usage section of readme](https://github.com/ashutosh1919/react-awesome-shapes#usage), you need to add new shape name and you need to link it with the `triangle.tsx`.

**Comment by judy-n** [comment]
@ashutosh1919 I just committed the changes 👍

**Comment by ashutosh1919** [comment]
@all-contributors please add @judy-n for code

**Comment by ashutosh1919** [comment]
Fixed #30

**Pull request merged**